DFXswiss/api
# Fees calculation and persistence, referencing assets, batch optimisation, monitoring improvements

### Release Checklist

#### Pre-Release

- [x] Check migrations
  - No database related infos (sqldb-xxx)
  - Impact on GS (new/removed columns)
- [x] Check for linter errors (in PR)
- [x] Test basic user operations
  - Login/logout
  - Registration
  - Update user data
  - Create/delete payment routes
- [x] turn off buy crypto before merge
- [x] check Asset table in production (to have all fee assets -> ETH, BNB, BTC Coins)
- [ ] After merge and migration -> rename DFI-Token to DFI
- [ ] After merge and migration complete -> run asset reference migration scripts in BuyCrypto, BuyCryptoBatch, LiquidityOrder and PayoutOrder

#### Post-Release

- Test basic user operations
- Monitor application insights log
